met-web/src/components/engagement/admin/view/ConfigSummary.tsx

```diff
@@ -178,6 +180,10 @@ export const ConfigSummary = () => {
                         </Grid>
                     </OutlineBox>
                 </Grid>
+                <Grid item>
+                    <WidgetPicker location={WidgetLocation.engagementAuthoring} />
```

Comment:
I assume in the future these location variables will be based on where the widget will render on the public view, rather than where it's edited on the backend

Reply:
Whichever makes the most sense to us, the devs. But probably, yes!